.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
     - CONDA_NPY=111  CONDA_PY=34
     - CONDA_NPY=110  CONDA_PY=35
     - CONDA_NPY=111  CONDA_PY=35
+    - CONDA_NPY=111  CONDA_PY=36
```

Comment:
This doesn't include NumPy 1.10 and 3.6, which is part of the reason I left PR ( https://github.com/conda-forge/matplotlib-feedstock/pull/97 ) open. However, it seems to have been closed. If you would rather not accept my PR, could you please fix this?